test/unit/controllers/todo.controller.test.ts

```diff
@@ -0,0 +1,130 @@
+import { expect } from '@loopback/testlab';
+import { TodoController } from '../../../src/controllers';
+import { TodoRepository } from '../../../src/repositories';
+import * as sinon from 'sinon';
+import { Todo } from '../../../src/models/index';
+
+describe('TodoController', () => {
+  let todoRepo: TodoRepository;
+
+  // Method stubs
+  let create: sinon.SinonStub;
+  let findById: sinon.SinonStub;
+  let find: sinon.SinonStub;
+  let replaceById: sinon.SinonStub;
+  let updateById: sinon.SinonStub;
+  let deleteById: sinon.SinonStub;
+
+  // Test fixtures
+  let controller: TodoController;
+  let todo: Todo;
+  let todoWithId: Todo;
+  let changedTodo: Todo;
+  let todoList: Todo[];
+
+  const noError = 'No error was thrown!';
+
+  describe('createTodo', () => {
+    beforeEach(resetRepositories);
```

Comment:
This hooks seems to be repeated in every second-level `describe`, can you move it to the first-level describe to avoid repetition?